pip/__init__.py

```diff
@@ -193,6 +193,17 @@ def check_isolated(args):
     return isolated
 
 
+def check_scripts_path(args):
+    if any(each in ['-t', '--target'] for each in args):
```

Comment:
All/most pip options can also be passed via configuration files and/or environmment variables.